drivers/interrupt_controller/vexriscv_litex.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019 Antmicro <www.antmicro.com>
```

Comment:
Is this copyright correct?

Reply:
Do you mean year? The WIP version of this code was indeed published in 2018, so I'll update this.

Reply:
I changed copyright year to 2018-2019 in every file.